nlp/2021_06_29_long_range_transformers/LongRangeTransformers.ipynb

```diff
@@ -0,0 +1,1940 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>big-bird</td>\n",
+       "      <td>128.060930</td>\n",
+       "      <td>711616</td>\n",
+       "      <td>2387.062487</td>\n",
+       "      <td>SHORT</td>\n",
+       "      <td>0.219696</td>\n",
+       "      <td>0.943508</td>\n",
+       "      <td>0.941891</td>\n",
+       "      <td>0.939446</td>\n",
+       "      <td>0.944348</td>\n",
+       "      <td>74.1597</td>\n",
+       "      <td>15.993</td>\n",
+       "      <td>4.005</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>512</td>\n",
+       "      <td>big-bird-512</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>roberta</td>\n",
+       "      <td>124.647170</td>\n",
+       "      <td>743692</td>\n",
+       "      <td>795.234547</td>\n",
+       "      <td>SHORT</td>\n",
+       "      <td>0.217354</td>\n",
+       "      <td>0.946880</td>\n",
+       "      <td>0.946200</td>\n",
+       "      <td>0.929530</td>\n",
+       "      <td>0.963478</td>\n",
+       "      <td>24.7230</td>\n",
+       "      <td>47.972</td>\n",
+       "      <td>12.013</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>512</td>\n",
+       "      <td>roberta-512</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>longformer</td>\n",
+       "      <td>148.660994</td>\n",
+       "      <td>839482</td>\n",
+       "      <td>2049.965436</td>\n",
+       "      <td>LONG</td>\n",
+       "      <td>0.371911</td>\n",
+       "      <td>0.923272</td>\n",
+       "      <td>0.921888</td>\n",
+       "      <td>0.917949</td>\n",
+       "      <td>0.925862</td>\n",
+       "      <td>68.2604</td>\n",
+       "      <td>17.375</td>\n",
+       "      <td>4.351</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>1024</td>\n",
+       "      <td>longformer-1024</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>big-bird</td>\n",
+       "      <td>128.060930</td>\n",
+       "      <td>711616</td>\n",
+       "      <td>2360.915053</td>\n",
+       "      <td>LONG</td>\n",
+       "      <td>0.280647</td>\n",
+       "      <td>0.923272</td>\n",
+       "      <td>0.920105</td>\n",
+       "      <td>0.937388</td>\n",
+       "      <td>0.903448</td>\n",
+       "      <td>75.4542</td>\n",
+       "      <td>15.718</td>\n",
+       "      <td>3.936</td>\n",
+       "      <td>5.0</td>\n",
+       "      <td>1024</td>\n",
+       "      <td>big-bird-1024</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>5</th>\n",
+       "      <td>roberta</td>\n",
+       "      <td>124.647170</td>\n",
+       "      <td>743692</td>\n",
+       "      <td>794.416961</td>\n",
+       "      <td>LONG</td>\n",
+       "      <td>0.485321</td>\n",
+       "      <td>0.899663</td>\n",
+       "      <td>0.900251</td>\n",
```

Comment:
Given the evolution in F1 score, do you feel the model is sufficiently trained?

Reply:
Not sure tbf, I think it somehow is, because the validation loss is increasing over the final iterations. Should I retrain with for example 10 epochs to validate?